### DISCUSSION TITLE
Device Authentication

### DISCUSSION THREAD

**Discussion by Neeraj-Nec** [edgexfoundry/community#211] [Q&A]
Can we apply strict device Identity Policies and token-based authentication on all the IoT devices which are connecting to Edgex.

**Comment by jiekechoo** [comment]
@Neeraj-Nec 
Can you explain more details about `all the IoT deices`?  The `southbound` device?

Jieke
YIQISOFT

**Reply by Neeraj-Nec** [reply]
Hi Jieke,

I am talking about all the IoT devices.

Thanks
Neeraj

**Reply by jiekechoo** [reply]
I'm so confused. I have two questions:
1. Are the devices EdgeX managed? And the devices connected to EdgeX or something else?
2. Do you use Identity Policies and token-based auth applied to devices or EdgeX?

**Reply by bnevis-i** [reply]
Also confused.  EdgeX is a middleware.  It doesn't know anything about a device other than what an EdgeX device service tells it.
